src/cascadia/TerminalSettingsEditor/LaunchViewModel.h

```diff
@@ -0,0 +1,33 @@
+// Copyright (c) Microsoft Corporation.
+// Licensed under the MIT license.
+
+#pragma once
+
+#include "LaunchViewModel.g.h"
+#include "ViewModelHelpers.h"
+#include "Utils.h"
+
+namespace winrt::Microsoft::Terminal::Settings::Editor::implementation
+{
+    struct LaunchViewModel : LaunchViewModelT<LaunchViewModel>, ViewModelHelper<LaunchViewModel>
+    {
+    public:
+        LaunchViewModel(Model::CascadiaSettings settings);
+
+        IInspectable CurrentDefaultProfile();
+        void CurrentDefaultProfile(const IInspectable& value);
+        winrt::Windows::Foundation::Collections::IObservableVector<IInspectable> DefaultProfiles() const;
+
+        WINRT_CALLBACK(PropertyChanged, Windows::UI::Xaml::Data::PropertyChangedEventHandler);
+

```

Comment:
When do we use this? I feel like we do because I see the that `DefaultProfiles` is observable, but we never call the property changed handler. Presumably, we should also make `CurrentDefaultProfile` observable? But again, we don't use it there.